Production/python/sampleConfig.py

```diff
@@ -3,11 +3,16 @@
 import sys
 from sets import Set
 import FWCore.ParameterSet.Config as cms
+import os
 
 from Configuration.Eras.Era_Run2_2016_cff import Run2_2016
 from Configuration.Eras.Era_Run2_2017_cff import Run2_2017
 from Configuration.Eras.Era_Run2_2018_cff import Run2_2018
-from Configuration.Eras.Era_Phase2C9_cff import Phase2C9
+
+# include Phase2 specific configuration only after 11_1_X
+cmssw_release_numbers = os.environ.get('CMSSW_VERSION').replace('CMSSW_','').split("_")
```

Comment:
I think it would be better if we avoid using CMSSW version in sampleConfig in order to simplify the long-term support (e.g. UL, Run3, other Phase2 geometries etc.).
What I meant before is to modify GetPeriodCfg in the following way:
```python
def GetPeriodCfg(sampleType):
    period = GetPeriod(sampleType)
    if period == 'Run2016':
        from Configuration.Eras.Era_Run2_2016_cff import Run2_2016
        return Run2_2016
    elif period == 'Run2017':
        from Configuration.Eras.Era_Run2_2017_cff import Run2_2017
        return Run2_2017
    elif period == 'Run2018':
        from Configuration.Eras.Era_Run2_2018_cff import Run2_2018
        return Run2_2018
    elif period == 'Phase2':
        from Configuration.Eras.Era_Phase2C9_cff import Phase2C9
        return Phase2C9
    else:
        raise RuntimeError('Period = "{}" is not supported.'.format(period))
```

Reply:
Ok, adjusted the method as you've suggested. Sorry for misunderstanding :)